home.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Beautiful Home Page</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+</head>
+<body>
+    <h1>Beautiful Home Page</h1>
```

Comment:
@amnaadev shouldn't this be a `h3` instead? What's the reason for `h1`?

Reply:
because h1 is cooler. Don't mess with my code. Just go sleep.